src/parser.yy

```diff
@@ -526,13 +527,16 @@ tuple_access_expr:
                 postfix_expr DOT INT      { $$ = driver.ctx.make_node<ast::FieldAccess>($1, $3, @3); }
                 ;
 
-
+block_expr:
+                "{" stmt_list expr "}" { $$ = driver.ctx.make_node<ast::Block>(std::move($2), $3, @$); }
```

Comment:
Should we also allow the final expression to be a call as well? Rust allows this.